app-arch/bzip2/bzip2-1.0.6.0.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic ltprune multilib-minimal
```

Comment:
*ltprune* is deprecated, see the header of [ltprune.eclass](https://github.com/gentoo/gentoo/blob/master/eclass/ltprune.eclass#L11).

Reply:
Oh ok. Forget who it was that suggested it, but someone in #gentoo-dev-help did.
They had an @ so I took it at face value.